k8s/wordpress/manifest/manifests.yaml.template

```diff
@@ -170,9 +176,26 @@ spec:
         - name: $APP_INSTANCE_NAME-wordpress-pvc
           mountPath: /var/www/html
           subPath: wp
+        lifecycle:
+          postStart:
+            exec:
+              command:
+              - "/bin/bash"
+              - "-euc"
+              - |
+                # https://codex.wordpress.org/Installing_WordPress#Installing_WordPress
+
+                curl --fail --retry 5 --retry-delay 5 --retry-connrefused \
+                  --data-urlencode "weblog_title=WordPress on Google Kubernetes Engine" \
+                  --data-urlencode "user_name=${WP_ADMIN_USERNAME}" \
+                  --data-urlencode "admin_email=noreply@google.com" \
+                  --data-urlencode "admin_password=${WP_ADMIN_PASSWORD}" \
+                  --data-urlencode "admin_password2=${WP_ADMIN_PASSWORD}" \
+                  --data-urlencode "pw_weak=1" \
```

Comment:
Provided password is generated by installation system. It should be stronger.

Reply:
A generated password has 12 chars. How much would you like to increase?